content/english/covid-distance-checker-wip/_index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "micro:bit — Wireless Social Distancing"
+description: "A guide to keeping at least 2 m apart with Bluetooth"
+date: 2020-07-297T08:45:08-07:00
+prereq: "None"
+difficulty: "Beginner"
+download: ""
+draft: true
+icon: "fas fa-microchip"
+weight: 1
+translationKey: "micro:bit-distancing"
+---
+## micro:bit - Wireless Social Distancing
```

Comment:
Usually we create different pages for each section (so that the pages aren't too long) can you consider doing this?